scripts/template/generate_transfers.py

```diff
@@ -0,0 +1,103 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Blockchain:
+    name: str
+    native_contract_address: str
+    wrapped_token_deposit: str
+    wrapped_token_withdrawal: str
+
+blockchains = [
+    Blockchain("arbitrum", "'0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'", None, None),
+    Blockchain("avalanche_c", None, None, None),
+    Blockchain("base", None, None, None),
```

Comment:
Do we need to add `bnb` or `bsc` blockchain here?

Also there are some token addresses we may use as reference in `tokens` table:
https://github.com/duneanalytics/spellbook/blob/4f9ff8c8399eef3b511a0226e442a493fabba603/models/tokens/tokens_native.sql